apps/payments/wise.py

```diff
@@ -203,53 +203,57 @@ def wise_business_profile():
     return id
 
 
+def _retrieve_detail(details, requested_type):
```

Comment:
Is there a reason why we can't make pywisetransfer return this as a dict (or even a dataclass) for us? Not sure what the API response looks like.

Reply:
(note also: that was gathered from the Wise Sandbox env)